ledger_validation_result.js

```diff
@@ -0,0 +1,45 @@
+const {AssetProof} = require('./asset_proof');
+
+/**
+ * To handle LedgerValidationResponse
+ */
+class LedgerValidationResult {
+  /**
+   * @param {StatusCode} code
+   * @param {AssetProof} proof
+   */
+  constructor(code, proof) {
+    this.code = code;
+    this.proof = proof;
+  }
+
+  /**
+   * @param {LedgerValidationResponse} response
```

Comment:
Let's make it consistent with other classes after we decide what to write here.
The same for the other classes.

Reply:
No problem !